portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,16 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomFunFact(){
+    const facts = 
+        ['Chris has a twin sister', 'Chris is trilingual', 'Chris knows Python'];
+    
+  //Pick a fact.
```

Comment:
nit: Let's add a space between the "//" and the comment, so it reads:

`// Pick a fact.`